WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PagesViewModel.kt

```diff
@@ -1047,6 +1075,37 @@ class PagesViewModel
         _editPage.postValue(Triple(site, result, loadAutoSaveRevision))
     }
 
+    private fun updateConflictedPostWithRemoteVersion(pageId: RemoteId) {
+        performIfNetworkAvailable {
+            val page = pageMap.getValue(pageId.value)
+            val post = postStore.getPostByLocalPostId(page.pageId)
+            if (post != null) {
+                post.error = null
+                post.setIsLocallyChanged(false)
+                post.setAutoSaveExcerpt(null)
+                post.setAutoSaveRevisionId(0)
+                dispatcher.dispatch(PostActionBuilder.newFetchPostAction(PostStore.RemotePostPayload(post, site)))
+                _showSnackbarMessage.postValue(
```

Comment:
if we show the snackbar at this time we are not sure that the newFetchPostAction has succeeded. What happens if it fails and we have already shown to the user the "Local version discarded"?

Reply:
I followed the same logic as posts here; the message is shown before the update is dispatched.  Take a look at `PostConflictResolver.updateConflictedPostWithLocalVersion` :(

Reply:
Yes, you are right about the `updateConflictedPostWithLocalVersion`. I guess in this case we use the "local" version that we already have in the device so it doesn't make much of a difference. In PostConflictResolver, when we call `updateConflictedPostWithRemoteVersion` and we make a call to fetch the remote post, we show the snackbar when the `conflictedPostUpdatedWithRemoteVersion()` callback is triggered. This will be called only on on success. I don't think that it is important but if it's easy to implement it would be nice to have for consistency. WDYT?